Gemfile

```diff
@@ -5,31 +5,14 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
I agree totally with the deletion of this kind of mindless boilerplate.

Can we split stuff like this into separate "tidy up" PRs (or at least commits)? It's not directly associated with adding `webpacker`, so it adds a bit of extra mental parse time. By no means a serious example, but since we're half in "koan" stage, now's the time to start thinking of being kind to future us, who are just more frazzled versions of "now" us :)